COMPONENTS.md

```diff
@@ -0,0 +1,160 @@
+# Form Components
+
+A set of form components provides support the the styled form elements in the CD Framework. 
+
+## Form
+
+The form component creates the required fieldset and legend.
+
+Use wire:submit on the form component rather than wire:click on the submit button to maintain the 
+expected behavior of submitting the form if the user hits enter in a text element.
+
+Here is an example of a simple form:
+
+```
+<x-cd.form.form legend="Simple form" wire:submit="submit">
+  <x-cd.form.text label="Name" wire:model="name"/>
+  <div>
+    <x-cd.form.submitbutton/>
+    <x-cd.form.resetbutton/>
+  </div>
+</x-cd.form.form>
+```
+
+## Form elements
+
+The `name` and `id` attributes on the form inputs default to the same value as 
+that used in the `wire:model` attribute. You may supply different values for `name` and `id  `.
+Inputs are not required by default; you can supply the `required="1"` attribute if the input
+should be required.
```

Comment:
```suggestion
Inputs are not required by default; you can supply the `:required="true"` attribute if the input
should be required.
```
How would you feel about this instead as the example code? I know they are functionally identical, but I think the readability is nicer and it avoids type coercion.

Reply:
I've made this change and also updated the handing of the required attribute to work even with required="false" which would otherwise give upexpected results.

I renamed the doc to FORM_COMPONENTS.md and linked it in the README.md.